File: Math/IsPrime/IsPrime.java
import java.lang.Math;

class IsPrime {
  public boolean isPrime(int n){
    int sqrt = (int) Math.sqrt(n);
    for(int i = 2; i <= sqrt; i++){
      if( n % i == 0){
        return false;
      }
    }
    return true;
  }

  public static void main(String[] args) {
    IsPrime ip = new IsPrime();
    System.out.println(ip.isPrime(18));
  }
}

File: PythonLeet/IsPrime/isprime.py
import math
def isPrime(num):
	isp = False
	
	i = 2
	while (i*i <= num):
		if (num % i == 0):
			isp = False	
			print("false")
			return isp
		else:
			isp = True 
		i = i+1

	if isp == True:
		print("true")
	return isp

isPrime(47)